### PR TITLE
Add Micronaut port

### PR DESCRIPTION
**Pull request by vasac**

### CODE REVIEW

File: java/micronaut-server/src/main/k8s/ingress.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+kind: Ingress
+metadata:
+  name: http-ingress
+spec:
+  rules:
+  - host: tasks.synoki.io
+    http:
+      paths:
+      - backend:
+          serviceName: todo-list-http
```
Comment (aseovic) [inline java/micronaut-server/src/main/k8s/ingress.yaml:17]: change to `todo-list-micronaut-server-http`

File: java/micronaut-server/src/main/k8s/ingress.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: grpc-ingress
+  annotations:
+    nginx.ingress.kubernetes.io/backend-protocol: "GRPC"
+spec:
+  rules:
+  - host: grpc.tasks.synoki.io
```
Comment (aseovic) [inline java/micronaut-server/src/main/k8s/ingress.yaml:28]: change to `grpc.tasks.micronaut.coherence.community`

File: java/micronaut-server/src/main/k8s/ingress.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    http:
+      paths:
+      - backend:
+          serviceName: todo-list-grpc
```
Comment (aseovic) [inline java/micronaut-server/src/main/k8s/ingress.yaml:32]: change to `todo-list-micronaut-server-grpc`

File: java/micronaut-server/src/main/resources/application.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+micronaut:
+  application:
+    name: todoListMicronautServer
```
Comment (aseovic) [inline java/micronaut-server/src/main/resources/application.yml:3]: change to `todo-list-micronaut-server`

File: java/micronaut-server/src/main/resources/application.yml [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+  server:
+    port: 3000
```
Comment (aseovic) [inline java/micronaut-server/src/main/resources/application.yml:5]: do you need to specify the port? isn't 3000 the default port?

File: java/micronaut-server/src/main/resources/logback.xml [hunk 6]
```diff
@@ -0,0 +1,15 @@
+<configuration>
+
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <withJansi>true</withJansi>
+        <!-- encoders are assigned the type
+             ch.qos.logback.classic.encoder.PatternLayoutEncoder by default -->
+        <encoder>
+            <pattern>%cyan(%d{HH:mm:ss.SSS}) %gray([%thread]) %highlight(%-5level) %magenta(%logger{36}) - %msg%n</pattern>
```
Comment (aseovic) [inline java/micronaut-server/src/main/resources/logback.xml:8]: Do we need to do any work to make Coherence log output look similar to standard Micronaut output?

We did that for Helidon...

**Review by aseovic — CHANGES_REQUESTED** [review]
Looks good overall, but there are a number of smaller changes that need to be made.

Also, please make sure that all the files in the PR use 2021 in the copyright headers.

File: java/micronaut-server/src/main/resources/application.yml (hunk 5 above)
Comment (vasac) [reply java/micronaut-server/src/main/resources/application.yml:5]: 8080 is the default Micronaut port.

**Review by aseovic — APPROVED** [review]

**Pull request merged**